### PR TITLE
Bug 1848492: Enforce 24h cookie expiration time

### PR DESCRIPTION
**Pull request by paulfantom**
* [ ] I added CHANGELOG entry for this change.
* [ ] No user facing changes, so no entry in CHANGELOG was needed.

By default, oauth-proxy expires cookie after 168 hours (7 days).

Fix is only for 3.11 as 4.x already has RFE in place.

/cc @s-urbaniak

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
+
 reviewers:
 - brancz
-- mxinden
-- elad661
-- ironcladlou
-- squat
 - s-urbaniak
+- paulfantom
+- LiliC
```
Comment (lilic) [inline OWNERS:7]: lower case :) lilic and thanks!

Comment (paulfantom) [reply OWNERS:7]: This is a copy-paste from [master](https://github.com/openshift/cluster-monitoring-operator/blob/master/OWNERS#L16) so you might want to fix this there too :)

Comment (lilic) [reply OWNERS]: Will do! Thanks!

**Pull request merged**